### PR TITLE
change the link title for what is taskcluster

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by djmitche** [comment]
The current link title is intended to answer the question on the previous line.  Any particular reason to change it?

**Comment by helfi92** [comment]
No reason other than the fact that I thought it was going to look cleaner and wanted to experiment with making my first pull request ever.

**Comment by djmitche** [comment]
Oh, hi Hassan -- I didn't recognize the github username.

Perhaps a better pull request would be to write a little bit of content for "I want to hack on TaskCluster itself" (maybe just talk about where to find the git repositories, making pull requests, etc.), and move the link to /tutorial/reviews onto that new page instead.  Want to give that a shot?

**Comment by helfi92** [comment]
Absolutely. I will start working on it first thing tomorrow

File: src/tutorial/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ operation, see the manual portion of this documentation.
 
 How do you want to use TaskCluster?
 
- * [I'm not sure - what is it??](tutorial/what-is-tc)
```
Comment (jonasfj) [inline src/tutorial/index.md]: @djmitche oh, I thought this was a "todo title" kinda of note... :) lol... I never read the question, just the bullet points, and they seemed weird, no wonder... :)

**Comment by helfi92** [comment]
@djmitche  with regards to your last message, i don't quite understand the following: "and move the link to /tutorial/reviews onto that new page instead"
did you mean move the link and content of /tutorial/review to the new link "I want to hack on TaskCluster itself" and remove "I want to submit changes of TaskCluster itself for review" ?

**Comment by djmitche** [comment]
Sorry I wasn't clear.  I was suggesting that the new page you create might link to `/tutorial/review`, and that the tutorial index page does *not* need to link to `/tutorial/review`.

**Pull request closed without merging**